.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -74,6 +74,10 @@ fun applyDefaultSettings(buildType: BuildType, os: OS = OS.linux, timeout: Int =
 
     buildType.requirements {
         contains("teamcity.agent.jvm.os.name", os.agentRequirement)
+        if (buildType.name.contains("Coordinator")) {
```

Comment:
I think it is better to add the following block to `PerformanceTest`:
```
    requirements {
        doesNotContain("teamcity.agent.name", "ec2")        
    }
```

WDYT?

Reply:
Absolutely better. Thanks.